cmd/ttnv3/application.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ttnv3
+
+import (
+	"github.com/spf13/cobra"
+	root "go.thethings.network/lorawan-stack-migrate/cmd"
+	"go.thethings.network/lorawan-stack-migrate/pkg/commands"
+	"go.thethings.network/lorawan-stack-migrate/pkg/source"
+)
+
+var applicationsCmd = &cobra.Command{
+	Use:     "application [app-id] ...",
+	Aliases: []string{"applications", "app"},
+	Short:   "Export all devices of an application",
+	Run: func(cmd *cobra.Command, args []string) {
+		commands.Export(cmd, args, func(s source.Source, item string) error {
+			return s.RangeDevices(item, root.ExportCfg.ExportDev)
```

Comment:
I would prefer that we parse `cmd.Flags().GetString("dev-id-prefix")` here directly than tying this to the root command package.

Reply:
How about we reference it from `cmd.Context()`?
https://github.com/TheThingsNetwork/lorawan-stack-migrate/pull/71/commits/27961386d6720750abeaa4cb12292e224957dadb